bindings/python/justfile

```diff
@@ -1,29 +1,35 @@
 set fallback
-VENV := env_var_or_default("VENV", "../../.venv")
+VENV := env_var_or_default("VENV", ".venv")
 VENV_BIN := VENV / "bin"
 
+VENV_DEV := env_var_or_default("VENV", ".venv-dev")
+VENV_DEV_BIN := VENV_DEV / "bin"
```

Comment:
why would we want two `.venv`'s? why not just put all of the dev dependencies into the same virtual env? Seems unnecessary, but maybe i'm missing something.

Reply:
So initially I had the `bindings/python` with their own `.venv`, but it was moved to the root _(not entirely sure why)_. Why not have a `bindings/python/.venv` and a `testing/.venv`? That is much more common & seems like it'd be the most sensible to me.

Reply:
I've collapsed into one venv for the bindings, but this comment is a bit out of date with the current state of the repo:

- there is a top-level `.venv` environment that installs poetry so that the pytests don't depend on system-poetry and don't have a bootstrapping problem. 
- `tests` has it's own venv with it's own dependencies. 
- `bindings/python` has it's own venv with it's own dependencies.